Make the calendar great again

MagneDavidsen opened the pull request.

ehamberg commented on package.json:
```diff
@@ -1,13 +1,25 @@
-    "test": "cd src/frontend && npm test"
+    "start": "webpack-dev-server",
+    "dist": "webpack",
+    "watch": "webpack --watch"    
```
Trailing whitespace

ehamberg commented on package.json:
```diff
@@ -1,13 +1,25 @@
   "author": "",
-  "license": "Proprietary"
+  "license": "UNLICENSED",
+  "dependencies": {
```
Many/most of these should be dev dependencies.

ehamberg commented:
The logic looks good.

A couple of (near-) show stoppers:

- You committed `src/dist/js/app.bundle.js` in `90c6789`
- Lots of dev dependencies included in build
- Quite a few uses of `var`, even in files where `const` (or `let`) is otherwise used
- Debugging printing left over

Almost all of these issues should be caught by something like eslint. Maybe you should base the eslint config on one of the other floq projects' so our es6 codebases are somewhat aligned. :)

MagneDavidsen commented:
@ehamberg 

- Oops, that's the old dist-folder that I removed from .gitignore. My bad! Removing it now.
- Yes, I'll clean it up later
- Yes, I'll do a es2015-lint cleanup in one commit later
- I can't see that I added any debug printing?

ehamberg commented:
Yep. Go-ahead for merging from me. The new stuff looks good. Let's do a clean-up after. :)
